config/config.py

```diff
@@ -10,14 +11,33 @@
 POSTED_EVENTS_FILE_PATH = "./data/posted_events.json"
 ONGOING_VOTES_FILE_PATH = "./data/ongoing_votes.json"
 
-
 # Load configuration
 config: configparser.ConfigParser = configparser.ConfigParser()
 config.read(CONFIG_ABSOLUTE_PATH)
 
+# Determine the environment
+ENV = os.getenv("ENV", "dev").lower()  # Default to 'dev' if ENV is not set
```

Comment:
why not just make values in "config.ini" uppercase?

Reply:
> why not just make values in "config.ini" uppercase?

Good point, will do.